Readme.md

```diff
@@ -0,0 +1,79 @@
+# elm-plots
+elm-plots is a plotting framework built entirely in elm using svgs. It draws inspiration from [d3](https://github.com/mbostock/d3) and [vega](https://github.com/vega/vega). Its goal is to make drawing visuals such as plots and charts easy to do. 
+
+## Plots
+A plot consists of top level attributes, scales, and visual elements. To make it easy to build up a plot, its functions are setup in a way to chain together updating and adding to it using the [forward function applicator](http://package.elm-lang.org/packages/elm-lang/core/3.0.0/Basics#|). For example a bar chart with vertical bars without axes can be created with the following elm code: 
```

Comment:
I recommend either rewording the beginning of this or renaming "top level attributes". Also, perhaps move the elm code example to after the descriptions of "top level attributes", "scales", and "visual elements".